streaming/BUILD.bazel

```diff
@@ -277,10 +282,13 @@ cc_binary(
         ".",  # needed for `include <jni.h>`
         "src",
     ],
+    copts = COPTS,
     linkshared = 1,
     linkstatic = 1,
     visibility = ["//visibility:public"],
     deps = [
+        "//:core_worker_lib",
+        "//:ray_util",
```

Comment:
`streaming_lib` depends on `core_worker_lib` and `ray_util`. Do we still need to add these here explicitly?

Reply:
Same as above; I was getting a linker error without it.